### PR TITLE
Created the talks page.

### PR DESCRIPTION
**Pull request by rafacianci**
related with #2 @fdaciuk @viniciusdacal can you review it?

PS: I created a JavaScript Function to show the speaker profile. 
PS²: The talks HTML is already made for mobile, but it's requiring a config on the header of the document.

### CODE REVIEW

**Comment by rafacianci** [comment]
And the files.. :sweat_smile: 

captura de tela 2016-03-13 as 12 49 24
captura de tela 2016-03-13 as 12 49 53

This is on a screen of 320px.
captura de tela 2016-03-13 as 12 51 02

**Comment by viniciusdacal** [comment]
:+1:

File: gulpfile.js [hunk 1]
```diff
@@ -41,9 +41,14 @@ gulp.task('stylus', () => {
 
 gulp.task('copy', () => {
   const imagesDest = './build/images'
+  const jsDest = './build/scripts'
   gulp.src('./src/images/**/*')
     .pipe(changed(imagesDest))
     .pipe(gulp.dest(imagesDest))
+
+  gulp.src('./src/scripts/*')
```
Comment (fdaciuk) [inline gulpfile.js]: -1
Crie uma nova task `js`. O `copy` é só para copiar as imagens.

**Comment by fdaciuk** [comment]
LGTM :+1:

File: gulpfile.js (hunk 1 above)
Comment (rafacianci) [inline gulpfile.js]: Done... :smile:

**Comment by fdaciuk** [comment]
Se tá testado, manda brasa no merge! :dancer:

**Comment by rafacianci** [comment]
Ahh eu e o @viniciusdacal estamos sem permissão pra dar merge.. :sweat_smile:

**Comment by fdaciuk** [comment]
Sério? Temos que ver isso! Vou fazer o merge aqui xD

**Pull request merged**